src/userAttrCache.h

```diff
@@ -22,13 +22,10 @@ namespace karere
  */
 enum {
     /** First attribute that is not directly supported by the SDK via getUserAttribute()*/
-    USER_ATTR_VIRTUAL_FIRST = 64,
-
-    /** RSA Public key of user */
-    USER_ATTR_RSA_PUBKEY = USER_ATTR_VIRTUAL_FIRST,
+    USER_ATTR_VIRTUAL_FIRST = 65,
```

Comment:
`64` was probably not a random value. Please, keep it and do not remove the value in the enum of callbacks. Simply mark the `USER_ATTR_RSA_PUBKEY` as deprecated, with a comment, so 64 simply becomes unused.

Reply:
This change it's related with this one:  db.query("delete from userattrs where type = 64");
Previously USER_ATTR_RSA_PUBKEY = USER_ATTR_VIRTUAL_FIRST = 64
We have removed all userattrs where type is 64 from cache to avoid to retrieve them.

Now  USER_ATTR_EMAIL = USER_ATTR_VIRTUAL_FIRST = 65

Reply:
Better to restore the 64, even if not used